app/controllers/sessions/edit_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Sessions::EditController < ApplicationController
+  before_action :set_session
+
+  def edit_send_consent_requests_at
+    render :send_consent_requests_at
+  end
+
+  def update_send_consent_requests_at
+    if !send_consent_requests_at_validator.date_params_valid?
+      @session.send_consent_requests_at =
+        send_consent_requests_at_validator.date_params_as_struct
+      render :send_consent_requests_at, status: :unprocessable_entity
+    elsif !@session.update(send_consent_requests_at_params)
+      render :send_consent_requests_at, status: :unprocessable_entity
+    else
+      redirect_to edit_session_path(@session)
+    end
+  end
+
+  def edit_weeks_before_consent_reminders
+    render :weeks_before_consent_reminders
+  end
+
+  def update_weeks_before_consent_reminders
+    Rails.logger.debug weeks_before_consent_reminders_params
```

Comment:
Left in deliberately?

Reply:
Nope... revealing my process of debugging there 😆